### DISCUSSION TITLE
need help finding type of chromebook

### DISCUSSION THREAD

**Discussion by regularmovie** [3kh0/ext-remover#981] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I need to downgrade but I don't know what type of Octopus version my computer is because there is multiple options under Octopus (sorry I'm a skid) these are the versions I'm talking about [source](https://chromiumdash.appspot.com/serving-builds?deviceCategory=ChromeOS

**Comment by nathblade16** [comment]
its under firmware version in chrome://version

**Reply by regularmovie** [reply]
its blocked

**Reply by AshtonDavies** [reply]
> its blocked
> 

L

**Reply by nathblade16** [reply, marked as answer]
it might show at the top of the diagnostic app, if not tell me your cb model

**Reply by v4pl** [reply]
lol

**Reply by nathblade16** [reply]
It worked I assume

**Reply by regularmovie** [reply]
@bloohacked  yes
